### PR TITLE
Throw error instead of logging when Qml constructor is not found

### PR DESCRIPTION
**Pull request by henrikrudstrom**
Im not a big fan of failing silently, is there any reason not to throw more errors?

### CODE REVIEW

**Comment by Plaristote** [comment]
I would say it might be useful in order to figure out whether a constructor is available or not. As long as we don't see an actual need for it, I see no reason not to merge this anyway. Though I think we should first get rid of line 159, it's unreachable now.

File: src/qtcore/qml/qml.js [hunk 1]
```diff
@@ -155,7 +155,7 @@ function construct(meta) {
                 item.dom.className += " " + meta.object.$class + (meta.object.id ? " " + meta.object.id : "");
             var dProp; // Handle default properties
         } else {
-            console.log("No constructor found for " + meta.object.$class);
+            throw Error("No constructor found for " + meta.object.$class);
```
Comment (ChALkeR) [inline src/qtcore/qml/qml.js]: `throw new Error`. Also, the `return` isn't needed.

**Comment by Plaristote** [comment]
I took the liberty of fixing both issues (`throw new Error` and the unreachable `return`).
That looks good to me now.

**Comment by ChALkeR** [comment]
LGTM

**Pull request merged**